src/boot/efi/assert.c

```diff
@@ -9,7 +9,7 @@
 void efi_assert(const char *expr, const char *file, unsigned line, const char *function) {
       log_error_stall(L"systemd-boot assertion '%a' failed at %a:%u, function %a(). Halting.", expr, file, line, function);
       for (;;)
-            uefi_call_wrapper(BS->Stall, 1, 60 * 1000 * 1000);
+            BS->Stall(60 * 1000 * 1000);
```

Comment:
oh, hmm, really? we can avoid this? are there docs that explain things about this somewhere? would be great to get rid of this